docs/guide/index.md

```diff
@@ -18,70 +22,110 @@ A-Frame is an open-source framework for creating 3D and virtual reality experien
+## Why A-Frame?
+
+A-Frame was built by the [Mozilla VR team][mozvr] to make it **quicker** and
+**easier** to build 3D/VR scenes in order for them to prototype faster and to
+bridge the web development community into the WebVR ecosystem. For WebVR to
+succeed, it needs content.  There are only a handful of WebGL developers in the
+world, but there are *millions* of web developers, designers, and artists.
+A-Frame puts 3D/VR content creation into the hands of everyone.
+
+### A-Frame Reduces Boilerplate
+
+Without A-Frame, starting a proper WebVR project is a lot of effort. You need
+to know what you are doing and repeat the same tedious work for every scene:
+
+Import the WebVR polyfill, look through the three.js repository examples for
+`VREffect`, declare a canvas, create a camera, set up lights, instantiate a
+render loop, wire everything together, build UI to enter VR, etc.
+
+Would toy sites like [zombo.com](https://zombo.com) exist if simply getting
+started required this much work? WebVR should thrive with long tail bite-sized
+experiences, but boilerplate is a strong barrier to motivation.
+
+In A-Frame, all boilerplate is reduced to one line of HTML: **`<a-scene>`**
+
+<video autoplay loop src="/videos/boilerplate.mp4"></video>
+
+All pieces of work in general are greatly reduced.  Rather than creating a
+mesh, creating a geometry, creating a material, and appending to scene...we can
+just add one of HTML.
+
+### A-Frame is Tailored for Web Developers
+
+HTML is arguably the easiest interface in all of computing, and it is
+universally understood amongst web developers and even non-developers. With
+A-Frame based on the DOM, we can manipulate scenes like we would with other web
+application: `getAttribute`, `setAttribute`, `querySelector`, etc., it's all
+there.
+
+This also means that most JavaScript frameworks and libraries integrate with
+A-Frame out-of-the-box. d3, React, Vue.js, Meteor, jQuery all work like a
+charm.
+
+A-Frame was built by front-end web developers *for* front-end web developers.
+Do you like using npm, GitHub, and the shiniest JavaScript frameworks? So do
+we.
+
+### A-Frame Provides Structure to three.js
+
+> "A-Frame is like when MVC landed in traditional front-end work...[where]
+three.js is like jQuery." &mdash; @wizgrav
+
+[three.js][three] is amazing. It has provided mere mortals that ability to work
+with 3D WebGL. Though like jQuery, three.js code is very loosely structured.
+This makes most three.js code hard to reuse and hard to copy-and-paste.
+
+**A-Frame is a declarative entity-component-system framework for three.js.**
+
+Developers can bundle three.js and JavaScript code within A-Frame components.
+These components can be reused and composed. If published and shared, these
+components can be used by other developers via the HTML abstraction layer.
```

Comment:
"via the HTML abstraction layer" sounds more complicated than the process of including and using a component